index.bs

```diff
@@ -1437,18 +1441,38 @@ object is discarded in the runtime, subsequent attempts to access it via the
 protocol will result in an error.
 
 <div algorithm>
-To get the <dfn>object id for an object</dfn> given |realm| and |object|:
+To get the <dfn>handle for an object</dfn> given |realm|, |ownership|
+and |object|:
+
+1. If |ownership| is equal <code>none</code>, return <code>null</code>.
+
+1. Let |handle id| be a new, unique, string handle for |object|.
+
+1. Set the value of |handle id| in |realm|'s [=handle object map=] to |object|.
+
+1. Return |handle id| as a result.
+
+</div>
+
+<div algorithm>
+To <dfn>set internal ids if needed</dfn> given |object remote value map|,
```

Comment:
I find the name "object remote value map" quite confusing. Couldn't it be "internal id map" or something similar?

Reply:
`|serialization internal map|`